core/capabilities/integration_tests/streams_test.go

```diff
@@ -0,0 +1,91 @@
+package integration_tests
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/datastreams"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/feeds_consumer"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
+)
+
+var (
+	workflowName    = "abcdef0123"
+	workflowOwnerID = "0100000000000000000000000000000000000001"
+)
+
+func Test_AllAtOnceTransmissionSchedule(t *testing.T) {
+	ctx := testutils.Context(t)
+
+	// The don IDs set in the below calls are inferred from the order in which the dons are added to the capabilities registry
+	// in the setupCapabilitiesRegistryContract function, should this order change the don IDs will need updating.
+	workflowDonInfo := createDonInfo(t, 1, 5, 1)
+	triggerDonInfo := createDonInfo(t, 2, 7, 1)
+	targetDonInfo := createDonInfo(t, 3, 4, 1)
```

Comment:
@ettec Maybe passing in a struct would be better here? That way we'd know what each of the ints is for: I assume it's donID, numNodes and F?

Reply:
sorted, ta